### DISCUSSION TITLE
Speech-to-text: Long numbers recognition

### DISCUSSION THREAD

**Discussion by adibo2025** [deepgram/community#914] [General help]
Hello,
I'm struggling with recognition of long identification numbers (e.g. 11 chars) - deepgram is returning only few numbers (often with mistakes) while Google speech-to-text V2 is getting 100% right results (with basic config). What parameters should I use to get it work?

Example number is: 
38245720119

Thanks!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@Aurero could you provide the information our bot is requesting above? ☝️  it will help us to troubleshoot.

**Comment by deepgram-community** [comment]
hey @aurero, any updates here?

> luke
